Probe/src/main/data_collector.py

```diff
@@ -6,6 +6,8 @@
 import json
 from dotenv import load_dotenv
 
+
+
 load_dotenv()
```

Comment:
Becouse we decided to make these configurable and has defaults, let's hardcode default values (e.g. `/tmp/survey6`) and replace those if ond only if new values are set from the environment.

You may hardcode them here, or I would suggest to move that to a seperate `config.py` and import it here.

First, let me know if this a quick fix. Otherwise, we can finalise this and do the config refactoring in a later PR.

Reply:
I think a new PR would be better: We'll follow the convention in both the probe and the server and update necessary changes everywhere required.